src/murmur/MurmurGRPCImpl.h

```diff
@@ -40,6 +40,7 @@ namespace MurmurRPC {
 class MurmurRPCImpl : public QThread {
 		Q_OBJECT;
 		std::unique_ptr<grpc::Server> m_server;
+		volatile bool b_IsRunning;
```

Comment:
Should be m_isRunning (see https://wiki.mumble.info/wiki/Coding_Guidelines#Member_variables)

Reply:
This is what I get for trying to remember the code style guidelines off the top of my head.